modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/raft/RocksDbClusterStateStorage.java

```diff
@@ -108,6 +108,22 @@ public void put(byte[] key, byte[] value) {
         }
     }
 
+    @Override
+    public void replaceAll(byte[] prefix, byte[] key, byte[] value) {
+        try (
+                var batch = new WriteBatch();
+                var options = new WriteOptions();
+        ) {
+            batch.deleteRange(prefix, RocksUtils.incrementArray(prefix));
```

Comment:
It seems that, if `prefix` consists of `0xFF` bytes only, `incrementArray()` will return `null` and `deleteRange()` will fail.

For instance, the prefix is {0xFF}, it contains key {0xFF, 0x5}, and we want to replace it with {0xFF, 0x10}.

Should a scan be added for such unfortunate prefixes?

Reply:
This is currently impossible, because we always have a prefix that is does not consist of 0xFFs. I can add an assertion if you like

Reply:
Yes, please, it would be great to have an assertion